var/spack/repos/builtin.mock/packages/openblas/package.py

```diff
@@ -25,4 +25,6 @@ class Openblas(Package):
     # To ensure test works with newer gcc versions
     conflicts("%gcc@:10.1", when="@0.2.16:")
 
+    depends_on("perl")
```

Comment:
Why we made the mock openblas depend on `perl`?

Reply:
I used it for an example of propagating a variant that is not in the source package but is in a 1st and 2nd level dependencies. It was just to make sure the variant was propagated properly.

Reply:
Ok, in that case I guess it would have been better to create another package with the relevant structure, and document briefly the graph/tree in the test docstring. 

That would document why we did what we did and, when some test fails in future PRs, would give some hint on the structure it was intended to test (so developers can check if anything changed in the meanwhile).

Reply:
Anyhow, not a change requested in this PR.